src/riak_core_util.erl

```diff
@@ -150,6 +155,55 @@ replace_file(FN, Data) ->
             Err
     end.
 
+%% @doc Similar to {@link file:consult/1} but will consult files that
+%% are not valid unicode. Specifially files written with
+%% `io_lib:format("~p,", io_list())' in r16 before `utf8' became the
+%% default encoding.
+-spec consult(file:filename()) -> {ok, [term()]} | {error, Err :: term()}.
+consult(FName) ->
+    case file:consult(FName) of
+        {ok, Terms} ->
+            {ok, Terms};
+        {error, _Reason} ->
+            latin_consult(FName)
```

Comment:
Shouldn't bother with this unless there's reason to believe it may work. file:consult will have used an encoding marker in the file if there was one, so we can assume there's not one there, or i's wrong :(, or the file's bad, or __maybe__ we're tripping over a default encoding mismatch.

This is getting too long, I'll put it in a gist and link it ...